addon/components/polaris-resource-list/bulk-actions.js

```diff
@@ -350,6 +338,18 @@ export default Component.extend(
       }
     ),
 
+    get moreActionsNode() {
```

Comment:
Does this syntax work with `EmberObject` class declarations? 😮 If it does, how do you feel about updating the other CPs in this file so we have consistency - or is that not something we can do since these are volatile and the others aren't?

Reply:
Chiming in 🙂

This [getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) is a js language functionality, so will work with any pojo.

[The ember deprecation for this](https://deprecations.emberjs.com/v3.x/#toc_computed-property-volatile) explains why this is a good idea for CPs marked volatile(TLDR, we invented volatile instead of recommending getters in the first place😅). In other cases we'll lose the advantage of accessing cached values if they were plain getters instead of CPs.

Reply:
Thanks for the explanation @sivakumar-kailasam! 🙇